backend/beyond_the_loop/models/stripe_payment_histories.py

```
@@ -0,0 +1,86 @@
+    """Service class for managing StripePaymentHistory records."""
+
+    def log_payment(self, payment_data: dict) -> Optional[StripePaymentHistory]:
+        """Log a new payment in the database."""
+        try:
+            with get_db() as db:
+                new_payment = StripePaymentHistory(**payment_data)
+                db.add(new_payment)
+                db.commit()
+                db.refresh(new_payment)
+                return new_payment
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)